src/SharedFrameworkValidation/RestoreSDKProject/RestoreSDKProject.csproj

```diff
@@ -61,4 +61,14 @@
     <Copy SourceFiles="%(ReferencePath.Identity)" DestinationFolder="$(_MetaPackageDestinationFolder)" />
   </Target>
 
+  <Target Name="CopyNETCoreAppVersionsFile" DependsOnTargets="ResolveLockFileCopyLocalProjectDeps" BeforeTargets="CoreCompile">
+    <PropertyGroup>
+        <NETCoreAppVersionsFilePath Condition="$([System.String]::new('%(FileDefinitions.Identity)').Contains('Microsoft.NETCore.App.versions.txt'))">%(FileDefinitions.ResolvedPath)</NETCoreAppVersionsFilePath>
```

Comment:
AFAIK, `dotnet build` does not yet support passing in a packages directory, so basically we would fail to find the assets we restored. Would you rather me restore the package twice so that one of those times it goes in the packages folder and we can restore?